actors/builtin/multisig/multisig_actor_state.go

```diff
@@ -55,38 +55,40 @@ func (st *MultiSigActorState) _hasAvailable(currBalance abi.TokenAmount, amountT
 	return true
 }
 
-func (as *MultiSigActorState) getPendingTransaction(rt vmr.Runtime, txnID TxnID) MultiSigTransaction {
+func (as *MultiSigActorState) getPendingTransaction(rt vmr.Runtime, txnID TxnID) (MultiSigTransaction, error) {
 	hm := adt.NewMap(adt.AsStore(rt), as.PendingTxns)
 
 	var out MultiSigTransaction
-	err := hm.Get(txnID, &out)
-	requireNoStateErr(rt, err, "failed to get transaction %v from pending transaction HAMT", txnID)
+	found, err := hm.Get(txnID, &out)
+	if err != nil {
+		return MultiSigTransaction{}, errors.Wrapf(err, "failed to read transaction %v from HAMT %s", txnID, as.PendingTxns)
```

Comment:
Both the key and the root are available inside the Map.Get, so I would put them in the inner message, then simplify this one to `failed to read transaction: `. General principle of including information at the lowest level it's available